parcel.go

```diff
@@ -14,47 +14,110 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt),
+	)
+	if err != nil {
+		return 0, err
+	}
+
+	lastID, _ := res.LastInsertId()
```

Comment:
Лучше проверять ошибку
```suggestion
	lastID, err := res.LastInsertId()
```